invenio_records_lom/templates/invenio_records_lom/record.html

```diff
@@ -11,128 +11,144 @@
-            data-tooltip="{{ record.ui.access_status.description_l10n }}">
-        {% if record.ui.access_status.icon %}
-        <i class="icon {{ record.ui.access_status.icon }}"></i>
-        {% endif %}
-        {{ record.ui.access_status.title_l10n }}
-      </span>
+  {% block record_header_button %}
+    {% if is_preview and not is_preview_submission_request and can_manage_record and is_draft %}
+      <nav class="back-navigation rel-pb-2 pl-0" aria-label="{{ _('Back-navigation') }}">
+        {% set back_page = url_for('invenio_records_lom.deposit_edit', pid_value=record.id) %}
+        {% set back_btn_label = _('Back to edit') %}
+        <a class="ui button labeled icon small compact" href="{{ back_page }}">
+          <i class="ui icon angle left"></i> {{ back_btn_label }}
+        </a>
+      </nav>
+    {% endif %}
+  {% endblock record_header_button %}
+
+  {# parent-template has record_management here, this isn't enabled for lom for now... #}
+
+  <section id="record-info" aria-label="{{ _('Labels for version, resource-type, access-status') }}">
+    <div class="ui grid middle aligned">
+      <div class="two column row">
+        <div class="left floated left aligned column">
+          {# parent-template has this here, but which date to show?
+            <span class="ui" title="{{ _('Publication date') }}">
+              {{ _('Published') }} {{ record.ui.publication_date_l10n_long }}
+            </span>
+          #}
+          {# version would only be an out-of-context number, could show this after implementing version side-card
+            <span class="label text-muted">
+              {{ _('Version {version_number}').format(version_number=record.ui.version) }}
+            </span>
+          #}
+        </div>
+        <div class="right floated right aligned column">
+          {% if record.ui.resource_type %}
+            <span class="ui label small neutral mb-5"
+                  title="{{ _('Resource type') }}">{{ record.ui.resource_type | capitalize }}</span>
+          {% endif %}
+          <span class="ui label small access-status {{ record.ui.access_status.id }} mb-5"
+                title="{{ _('Access status') }}"
+                data-tooltip="{{ record.ui.access_status.description_l10n }}"
+                data-inverted=""
+          >
+            {% if record.ui.access_status.icon %}
+              <i class="icon {{ record.ui.access_status.icon }}"></i>
+            {% endif %}
+            {{ record.ui.access_status.title_l10n }}
+          </span>
+        </div>
+      </div>
     </div>
-  </div>
-</div>
+  </section>
+  <div class="ui divider hidden"></div>
 {% endblock record_header %}
 
+{# Title #}
 {% block record_title %}
-<h1>{{ record.ui.title }}</h1>
+  <section id="record-title-section" aria-label="{{ _('Record title and creators') }}">
+    <h1 id="record-title" class="wrap-overflowing-text">{{ record.ui.title }}</h1>
+    <section id="creatibutors" aria-label="{{ _('Contributors') }}">
```

Comment:
minor: is section the correct html-tag here? i thought that section should always have a h* tag? Nested section may be allowed but seams strange to me.

Reply:
parent-template does something similar, but it might be bad there too: [parent-template](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html#L205)
frankly, html-elements and their usage ain't my strongest suit
open for any suggestions on this...

Reply:
i would use the section-tag only with a large context. Base rule not nest section. Do it only if there are good points for it. A good point would be to have another title tag also in the nested section, which is not the case here. Also not in your mentioned example.